homework/index.js

```diff
@@ -30,21 +30,53 @@
   }
 
   function renderRepoDetails(repo, ul) {
-    createAndAppend('li', ul, { text: repo.name });
+    createAndAppend('p', ul, { text: `Repository:` });
+    createAndAppend('p', ul, { text: `Description:` });
+    createAndAppend('p', ul, { text: `Forks:` });
+    createAndAppend('p', ul, { text: `Updated:` });
+  }
+
+  function renderRepoDetailsTitle(repo, ul) {
+    createAndAppend('a', ul, {
+      text: repo.name,
+      href: repo.clone_url,
+      target: '_blank',
+    });
+    createAndAppend('p', ul, { text: repo.description });
+    createAndAppend('p', ul, { text: repo.forks });
+    createAndAppend('p', ul, { text: repo.updated_at });
+  }
+  function sortName(a, b) {
+    return a.name.localeCompare(b.name);
   }
 
   function main(url) {
     fetchJSON(url, (err, repos) => {
       const root = document.getElementById('root');
+      createAndAppend('div', root, {
+        text: 'HYF Repositories',
+        class: 'hyfTitle',
+      });
+
       if (err) {
         createAndAppend('div', root, {
           text: err.message,
           class: 'alert-error',
         });
         return;
       }
-      const ul = createAndAppend('ul', root);
-      repos.forEach(repo => renderRepoDetails(repo, ul));
+
+      repos.sort(sortName);
+      for (let i = 0; i < repos.length; i++) {
+        const div = createAndAppend('div', root);
+        const insideDiv = createAndAppend('div', div);
+        const insideDiv2 = createAndAppend('div', div);
```

Comment:
Avoid names that only differ by a meaningless number suffix. Try and invent more descriptive names.